rs/ledger_suite/icp/ledger/tests/tests.rs

```diff
@@ -1246,96 +1254,66 @@ fn test_upgrade_serialization() {
         upgrade_args,
         minter,
         false,
-        false,
     );
 }
 
 #[test]
-fn test_upgrade_serialization_fixed_tx() {
```

Comment:
`test_upgrade_serialization_fixed_tx` is replaced by `test_multi_step_migration` (which was renamed from `test_upgrade_serialization_fixed_tx`).